src/CoreLib/deploy.fs

```diff
@@ -256,9 +258,27 @@ type DeploymentSettings() =
     /// Maximum combined buffer devoted to sending queue (in bytes) 
     static member val MaxSendingQueueLimit = 1024 * 1024 * 50 with get, set
     /// Maximum network stack memory (total across all connections), 0 means unbounded (in bytes)
-    static member val MaxNetworkStackMemory = 1024 * 1024 * 1024
+    static member val MaxNetworkStackMemory = 1024 * 1024 * 1024 * 4 with get, set
```

Comment:
Is this a good default for general use or it's optimized towards sort benchmark?